src/recon_trace.erl

```
@@ -449,6 +466,13 @@ validate_tspec(Mod, Fun, Args) ->
         _ when Args >= 0, Args =< 255 -> {Args, true}
     end.
 
+validate_output_to(Opts) ->
+  OutputTo = proplists:get_value(output, Opts, undefined),
+  case OutputTo of
+    P when is_pid(OutputTo) -> P;
+    _ -> group_leader()
+  end.
```

the output could arguably go to a name, not just a pid, so atoms other than `undefined` could be accepted. How about
```erlang
validate_io_server(Opts) ->
    P = proplists:get_value(io_server, Opts, group_leader()).
```
And let it crash if it goes wrong.